src/views/Manager/index.js

```diff
@@ -32,7 +31,7 @@ const validationSchema = Yup.object().shape({
 const getManager = (context, managerId, storeInState) => {
   axios
     .get(`${process.env.REACT_APP_PROXY}/api/user/${managerId}`,
-    { Authorization: `Bearer ${context.user.accessJwt}` }
+      { Authorization: `Bearer ${context.user.accessJwt}` }
```

Comment:
Authorization object needs to be associated with a header key:
`{ headers: { 'Authorization': `Bearer ${context.user.accessJwt}` } }`
For a DRY way to do this, check out some of the files that use the `makeAuthHeaders` helper function

Reply:
I'm a BIG fan of this.  I didn't see it before.  Thank you!